ui/react-components/SingleDatePicker.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState, Fragment } from 'react'
+import { subDays } from "date-fns"
+import { addDays } from "date-fns"
```

Comment:
we are using moment, can you use the same?

Reply:
'ui/react-components/SingleDatePicker.jsx' should be in 'ui/react-components/components/SingleDatePicker.jsx'